packages/workspace/src/command-line/shared.ts

```diff
@@ -39,7 +39,7 @@ export function printArgsWarning(options: NxArgs) {
   }
 }
 
-export function parseFiles(options: NxArgs): { files: string[] } {
+export function parseFiles(options: yargs.Arguments): { files: string[] } {
```

Comment:
Why this change?

Reply:
We should only use the Arguments type when talking about executing tasks. The analysis part should not use Arguments.